test/check-session

```diff
@@ -34,8 +34,7 @@ class TestSession(MachineCase):
 
         def wait_session(should_exist):
             def cond():
-                check = check_in if should_exist else check_not_in
-                return check(m.execute("loginctl list-sessions"), "admin")
+                return should_exist == ("admin" in m.execute("loginctl list-sessions"))
```

Comment:
The original code would raise an exception in case of failure, while the replacement only returns the current state. Since we wrap `cond` in a `wait`-call, I believe this is ok.
Was the change intended?